README.md

```diff
@@ -194,30 +194,34 @@ Command to run a single test script locally using Ava:
 
 For the integration tests to work, you have to first run the localstack application, launch the localAPI and serve the dashboard first. Run the following commands in separate terminal sessions:
 
-Run background localstack application.
+Run background localstack application:
 ```bash
-  $ npm run start-localstack
+  npm run start-localstack
+```
+Note: you can also stop the localstack application as follows:
+```bash
+  npm run stop-localstack
 ```
 
 Serve the cumulus API (separate terminal)
 ```bash
-  $ npm run serve-api
+  npm run serve-api
 ```
 
 Serve the dashboard web application (another terminal)
 ```bash
-  $ [HIDE_PDR=false ENABLE_RECOVERY=true APIROOT=http://localhost:5001] npm run serve
+  [HIDE_PDR=false ENABLE_RECOVERY=true APIROOT=http://localhost:5001] npm run serve
 ```
 
 If you're just testing dashboard code, you can generally run all of the above commands as a single docker-compose stack.
 ```bash
-  $ npm run start-dashboard
+  npm run start-dashboard
 ```
 This brings up LocalStack, the Cumulus localAPI, and the dashboard.
 
 Run the test suite (yet another terminal window)
 ```bash
-  $ npm run cypress
+  npm run cypress
 ```
 
 When the cypress editor opens, click on `run all specs`.
```

Comment:
I'm showing 'Run 13 integration specs' in my Cypress editor window --  not sure if yours is different. 

<img width="1034" alt="Screenshot 2025-01-09 at 9 28 41 AM" src="https://github.com/user-attachments/assets/53e3d4ab-617f-4991-9bb1-4a73252c061d" />

Also, I would suggest adding to the docs that the browser type be 'Electron 93' to get the full interactive tests.

Reply:
Ha! I didn't change that. The only thing i changed was the bash scripts so they are copyable. Your documentation is broken! I can fix it though.